### PR TITLE
Topbar

### PR DESCRIPTION
**Pull request by rodrigoWolox**
## Summary
Topbar added with some navigation on it and the option of logout from the game.

## Screenshot
topbar

## Trello Card
https://trello.com/c/t6kVOKsr/25-routing-ii-topbar

### CODE REVIEW

File: rh-tictactoe/src/app/components/Topbar/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+
+import { actionCreator } from '../../../../src/redux/topbar/actions.js';
+
+import Styles from './styles.scss';
+
+class Links extends React.Component {
```
Comment (guidoprinc) [inline rh-tictactoe/src/app/components/Topbar/index.js]: function component

File: rh-tictactoe/src/app/screens/Stats/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class Stats extends React.Component {
+  render() {
+    return <h1>Stats</h1>;
+  }
+}
```
Comment (guidoprinc) [inline rh-tictactoe/src/app/screens/Stats/index.js]: function component

File: rh-tictactoe/src/redux/topbar/actions.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { cleanInfo } from '../../services/userService';
+
+export const actions = {
+  LOG_OUT: 'LOG_OUT'
+};
+
+export const actionCreator = {
+  logOut: dispatch => {
```
Comment (guidoprinc) [inline rh-tictactoe/src/redux/topbar/actions.js]: i would not create another section in redux for the topbar, just leave it as it was before

File: rh-tictactoe/src/app/components/Topbar/index.js [hunk 4]
```diff
@@ -3,26 +3,24 @@ import PropTypes from 'prop-types';
...
-        <NavLink className={Styles.link} to="/app/game">
-          TicTacToe
-        </NavLink>
-        <NavLink className={Styles.link} to="/app/stats">
-          Stats
-        </NavLink>
-      </nav>
-    );
-  }
+function Links(props) {
```
Comment (guidoprinc) [inline rh-tictactoe/src/app/components/Topbar/index.js]: when i mean function component is always in arrow form and destructing props values

File: rh-tictactoe/src/app/screens/Stats/index.js [hunk 5]
```diff
@@ -1,9 +1,7 @@
 import React from 'react';
 
-class Stats extends React.Component {
-  render() {
-    return <h1>Stats</h1>;
-  }
+function Stats() {
```
Comment (guidoprinc) [inline rh-tictactoe/src/app/screens/Stats/index.js]: same

File: rh-tictactoe/src/app/components/Topbar/index.js [hunk 6]
```diff
@@ -7,7 +7,7 @@ import { actionCreator } from '../../../../src/redux/game/actions.js';
 
 import Styles from './styles.scss';
 
-function Links(props) {
+const Links = (props) => {
```
[the remark this answers is not in the document]
Comment (rodrigoWolox) [reply rh-tictactoe/src/app/components/Topbar/index.js]: I deserve the bullet

**Pull request merged**